isis/tests/FunctionalTestsChan1m32isis.cpp

```diff
@@ -0,0 +1,558 @@
+#include <QTemporaryDir>
+
+#include "chan1m32isis.h"
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "Histogram.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using namespace Isis;
+using ::testing::HasSubstr;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/chan1m32isis.xml").expanded();
+
+TEST(Chan1m32Isis, Chan1m32IsisTestFowardAscending) {
+  QTemporaryDir prefix;
+  QString cubeFileName = prefix.path() + "/chan1m32isisTEMP.cub";
+  QVector<QString> args = {"from=data/chan1m32isis/forwardAscending/M3T20090630T083407_V03_L1B_cropped.LBL",
+                           "loc=data/chan1m32isis/forwardAscending/M3T20090630T083407_V03_LOC_cropped.IMG",
+                           "obs=data/chan1m32isis/forwardAscending/M3T20090630T083407_V03_OBS_cropped.IMG",
+                           "to=" + cubeFileName };
+
+  UserInterface options(APP_XML, args);
+  try {
+    chan1m32isis(options);
+  }
+  catch (IException &e) {
+    FAIL() << "Unable to ingest Chandrayaan image: " << e.toString().toStdString().c_str() << std::endl;
+  }
+  Cube cube(cubeFileName);
+  Pvl *isisLabel = cube.label();
+
+  // Dimensions Group
+  PvlGroup &dimensions = isisLabel->findGroup("Dimensions", Pvl::Traverse);
+  ASSERT_EQ(int(dimensions["Samples"]), 608);
+  ASSERT_EQ(int(dimensions["Lines"]), 5);
+  ASSERT_EQ(int(dimensions["Bands"]), 3);
+
+  // Pixels Group
+  PvlGroup &pixels = isisLabel->findGroup("Pixels", Pvl::Traverse);
+  ASSERT_EQ(pixels["Type"][0].toStdString(), "Real");
+  ASSERT_EQ(pixels["ByteOrder"][0].toStdString(), "Lsb");
+  ASSERT_EQ(double(pixels["Base"]), 0.0);
+  ASSERT_EQ(double(pixels["Multiplier"]), 1.0);
```

Comment:
Another comment made on another 2isis PR, these values can be accessed directly from functions that exist on the cube. 

`sample/line/bandcount` for the number of samples, lines, and bands
`pixelType, bandOrder, base, and multiplier` for the latter group